### PR TITLE
Repl

### PR DESCRIPTION
**Pull request by nicktate**
Made a little myriad-kv repl. Use the standard commands, supports auto complete, json formatting, and some super basic colorization. 

Set up main application to launch auto into repl if it is run directly from command line rather than required so you can just `npm install -g` the client and it should work.

I have to look into some stuff... auto-complete may only work at node 6.4. I think I got un-lucky and the option I happened to be using was added in like the most recent version of node. I'll figure out how to add legacy support if that is the case. 
@normanjoyner

### CODE REVIEW

**Comment by nicktate** [comment]
image

File: repl.js [hunk 1]
```diff
@@ -0,0 +1,207 @@
+'use strict';
+
+const MyriadKV = require('./application');
+const mc = new MyriadKV({
+    host: '0.0.0.0',
```
Comment (normanjoyner) [inline repl.js]: we'll need a way to override this with a command line flag (either specifying interface or ip address). same with port

File: package.json [hunk 2]
```diff
@@ -22,7 +23,10 @@
   },
   "homepage": "https://github.com/normanjoyner/node-myriad-kv-client",
   "dependencies": {
-    "lodash": "^4.5.0",
-    "mocha": "^2.4.5"
+    "lodash": "^4.15.0",
+    "mocha": "^3.0.2"
```
Comment (normanjoyner) [inline package.json]: change to a dev dependency

**Comment by nicktate** [comment]
@normanjoyner updated.

**Pull request merged**